set_alt: 0.8
pos_k_p_x: 1
pos_k_p_y: 1
pos_k_i: 0 
vel_x_k_p: 0.17
vel_x_k_i: 0 
vel_x_k_d: 0 
vel_y_k_p: 0.17
vel_y_k_i: 0
vel_y_k_d: 0
yaw_reset: 1
